### ISSUE TITLE
Documentation should mention quantities (grootheden) of parameters

### ISSUE THREAD

**Issue by Huite** [Deltares/QGIS-Tim#138]
Coming from a question that was e-mailed to me: the TimML API reference currently does not mention what the units or quantities are. Not having units is a feature, just like MODFLOW, as some people want to compute in m/d, ft/d, m/s, etc.
This is worth adding to TimML (and TTim), but also to the QGIS-Tim Docs (that admittedly need a lot of other work too...).

It would be worthwhile to add the quantities like MODFLOW does in the documentation, e.g. L/T for length over time, L^3 for volumes, etc.

**Comment by JoerivanEngelen** [comment]
**Note:**
Not having units holds true for the TimML and TTIM docs, but not for QGIS-Tim.

Lengths between x,y locations are determined based on the CRS. These have a unit (e.g. meters for most projected ones). 
[There's even a check if a CRS is used which uses meters or feet](https://github.com/Deltares/QGIS-Tim/blob/fdc4ae30981683c8e88423268aaae6d1cee62984/plugin/qgistim/widgets/dataset_widget.py#L425)

More explanation on quantities is good, e.g. whether a parameters is in L^3/T or L/T, but for QGIS-Tim you can already reduce L^3/T to: m^3/T or feet^3/T

**Comment by Huite** [comment]
Good point. The time unit is also implicitly defined when writing TTim output: the float-based time is interpreted as (decimal) days since the reference date.
